### PR TITLE
ACS-8768: wait with processing till acs will be healthy

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

**Review by mpichura — APPROVED** [review]
LGTM - have a look at my suggestion.

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/config/setup/ProcessingStarter.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.apache.camel.CamelContext;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import org.alfresco.hxi_connector.common.adapters.messaging.repository.AcsHealthProbe.AcsHealthy;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class ProcessingStarter
```
[the remark this answers is not in the document]
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/config/setup/ProcessingStarter.java]: done!

**Review by krdabrowski — APPROVED** [review]
LGTM

**Review by tpage-alfresco — APPROVED** [review]
LGTM

**Review by mpichura — APPROVED** [review]
LGTM

**Review by krdabrowski — APPROVED** [review]

**Pull request merged**